pkg/api/v1alpha2/capsule_types.go

```diff
@@ -227,6 +227,50 @@ type CapsuleInterface struct {
 
 	// Public specifies if and how the interface should be published.
 	Public *CapsulePublicInterface `json:"public,omitempty"`
+
+	// Host routes that are mapped to this interface.
+	Routes []HostRoute `json:"routes,omitempty"`
```

Comment:
How does this interact with the Ingress property of `Public`? Specifying a list of Routes with hosts means the corresponding ingress objects must exist to accept traffic from those hosts, right?

Reply:
Yep, exactly - though it's implementation specific how